.github/workflows/build-rust-injector.yml

```diff
@@ -22,6 +22,7 @@ jobs:
         run: |
           curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y --no-modify-path
           echo "$HOME/.cargo/bin" >> $GITHUB_PATH
+          sudo apt install build-essential -y
```

Comment:
This is already installed, was there a reason it needed to be installed again?  https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2204-Readme.md#installed-apt-packages

Reply:
no, but its explicit so that others reading the pipeline and attempting to replicate on their local machine can more easily do so